css-brochure-form/styles.css

```diff
@@ -0,0 +1,85 @@
+h2,
+label,
+button {
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  text-transform: uppercase;
+}
+
+.input-text,
+#number-of-engineers {
+  font-family: 'AtlasGrotesk', sans-serif;
+  font-style: normal;
+}
+
+h2 {
+  color: rgba(250, 65, 65, 0.808);
+  font-size: medium;
+  letter-spacing: 1.5px;
+}
+
+h2,
+form {
+  text-align: center;
+}
+
+input {
+  width: 42%;
+  height: 2.5rem;
+  border-radius: 7px;
+  border: thin solid rgba(145, 145, 145, 0.377);
+  padding-left: 10px;
+  color: rgba(145, 145, 145, 0.377);
+  margin-bottom: 15px;
+}
+
+#label-email {
+  font-size: 8pt;
+  color: black;
+  font-weight: 500;
+  margin-right: 28em;
+}
+
+#label-name {
+  font-size: 8pt;
+  color: black;
+  font-weight: 500;
+  margin-right: 33em;
+}
+
+#label-location {
+  font-size: 8pt;
+  color: black;
+  font-weight: 500;
+  margin-right: 24em;
+}
+
+#label-select {
+  font-size: 8pt;
+  color: black;
+  font-weight: 500;
+  margin-right: 5em;
+}
+
+select {
+  width: 42%;
+  height: 2.5rem;
+  border-radius: 7px;
+  border: thin solid rgba(145, 145, 145, 0.377);
+  padding-left: 2px;
+  color: black;
+  background-color: rgba(211, 211, 211, 0.24);
+  margin-bottom: 10px;
+  font-weight: 500;
+}
+
+button {
+  background-color: rgba(250, 65, 65, 0.808);
+  color: white;
+  border-radius: 20px;
+  width: 90px;
+  height: 33px;
+  border: thin solid rgba(145, 145, 145, 0.377);
+  font-weight: 500;
+  margin-left: 32%;
```

Comment:
Same thing here, lots of margin. You can push the button to the right of the form with the `text-align` property.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align